test/general/__init__.py

```diff
@@ -31,8 +33,11 @@ def setup_solo_multiworld(
     return setup_multiworld(world_type, steps, seed)
 
 
-def setup_multiworld(worlds: Union[List[Type[World]], Type[World]], steps: Tuple[str, ...] = gen_steps,
-                     seed: Optional[int] = None) -> MultiWorld:
+def setup_multiworld(
+        worlds: list[type[World[Region, Entrance, Location, Item]]] | type[World[Region, Entrance, Location, Item]],
```

Comment:
With the `TypeVar` default, this and other places can be changed back to `List[Type[World]] | Type[World]`

Reply:
huge